### PR TITLE
Add UI to Web3 provider

### PR DESCRIPTION
**Pull request by Marik-D**
# Summary
Contains #941 

Fixes #number

## Checklist
- [ ] Change is small and easy to review (please split big changes into multiple PRs)
- [ ] Change is consistent with architecture
- [ ] Change is consistent with test architecture
- [ ] Change is consistent with naming conventions
- [ ] New code is covered with tests
- [ ] Tests related to old code are updated
- [ ] Documentation is up to date with changes

### CODE REVIEW

File: universal-login-react/src/config/config.ts [hunk 1]
```diff
@@ -2,9 +2,9 @@ require('dotenv').config();
 import {ETHER_NATIVE_TOKEN} from '@universal-login/commons';
 
 const developmentConfig = {
-  domains: ['mylogin.eth'],
-  relayerUrl: 'http://localhost:3311',
-  jsonRpcUrl: 'http://localhost:18545',
+  domains: ['poppularapp.test'],
+  relayerUrl: 'https://relayer-rinkeby.herokuapp.com',
+  jsonRpcUrl: 'https://rinkeby.infura.io/v3/b3026fc5137a4bd18e5d5906ed49f77d',
```
Comment (marekkirejczyk) [inline universal-login-react/src/config/config.ts]: This should work by default on local host

Comment (Marik-D) [reply universal-login-react/src/config/config.ts]: Yeah, that was not supposed to be commited

File: universal-login-web3/lib/ui/App.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      Create or connect account
+    </p>
+    <p style={{ color: 'white', fontSize: 15 }}>
+      Type a nickname you want
+    </p>
+    <Onboarding
+      sdk={sdk}
+      onConnect={() => {}}
+      onCreate={onCreate}
+      domains={domains}
```
Comment (marekkirejczyk) [inline universal-login-web3/lib/ui/App.tsx]: Should we use a designated div instead in the future?

Comment (Marik-D) [reply universal-login-web3/lib/ui/App.tsx]: Absolutely, this will also get wrapped in some navigation for other UL features to be displayed

**Pull request merged**